### ISSUE TITLE
done <<< "$ET_SERVER_COMMANDS"

### ISSUE THREAD

**Issue by cjthompson** [MisterTea/EternalTCP#25]
When I try to connect to a remote Ubuntu server, I always get
```
Expected a string, but instead found a redirection
fish: done <<< "$ET_SERVER_COMMANDS"
```

I installed it on my Mac view Homebrew.  The remote server is Ubuntu 16.04 (armhf).  There are no pre-built arm binaries, so I cloned the Git repo and ran cmake / make / make install.

**Comment by MisterTea** [comment]
It looks like fish isn't handling the launcher correctly. Try using bash:

bash et ...

**Comment by cjthompson** [comment]
I tried it within a bash shell:
```
[04/11 11:13:51 AM] ~  [1.29] > bash
bash-3.2$ et sw.batterybarpro.com
chris@sw.batterybarpro.com's password:
Expected a string, but instead found a redirection
fish: done <<< "$ET_SERVER_COMMANDS"
```

and `bash et ....`

The remote server is also running fish as my default shell.  The script starts with `#!/bin/bash`, so I don't understand why it would even be trying to run fish.

**Comment by MisterTea** [comment]
Ah yeah the server must be running the script as fish. Can you change the
launcher so the server command also has a bash in front of it?

**Comment by cjthompson** [comment]
I was able to get it running like this:
```
et --ssh="-t example.com '/bin/bash'" example.com
```

**Comment by MisterTea** [comment]
Can you submit a pull request with that change?  Thanks!

**Comment by MisterTea** [comment]
Fixed by 8dfa5893037c67136994fe067f4ff6bfca052080
